WeatherTodayUITests/Sources/Specs/Application/ApplicationSpec.swift

```diff
@@ -1,4 +1,4 @@
-import Foundation
+// swiftlint:disable closure_body_length function_body_length
```

Comment:
Not sure why this file is missing the copyright labels. I found they are useful when seeing the owner.
```
//  Created by Taher on 20/10/22.
//  Copyright © 2022 Nimble. All rights reserved.
//
 ```

Reply:
This file is created by iOS template may be there are some problems when I ran scripts to generate files.